Dockerfile

```diff
@@ -1,72 +1,57 @@
-# ============== build stage ==================
-FROM debian as builder
+FROM debian:buster-20201117
```

Comment:
default is 10, just checked with a simple dockerfile

```dockerfile
FROM debian

CMD [ "cat" , "/etc/issue", "/etc/debian_version"]
```
`docker build . -t debtest && docker run -it debtest`

```log
Debian GNU/Linux 10 \n \l

10.6
```

Reply:
Yeah, I've got a strange reference in dockerhub (sent me to 9). This is a draft, I am testing right now :). I'll change it to full PR once I've got it ready to be reviewed ^^